mmdeploy/backend/snpe/backend_manager.py

```diff
@@ -33,3 +36,50 @@ def build_wrapper(cls,
             uri = kwargs['uri']
         return SNPEWrapper(
             dlc_file=backend_files[0], uri=uri, output_names=output_names)
+
+    @classmethod
+    def to_backend(cls,
```

Comment:
`deploy_cfg` was not used. Making unified interfaces introduced unwanted arguments.